core/templates/domain/topic_viewer/topic-viewer-backend-api.service.ts

```diff
@@ -33,6 +38,8 @@ export class TopicViewerBackendApiService {
     private http: HttpClient,
     private urlInterpolation: UrlInterpolationService) {}
 
+  private readOnlyTopicObjectFactory: ReadOnlyTopicObjectFactory = null;
```

Comment:
Hm, should the variables below also have a type?

Reply:
Later it's declared with `ReadOnlyTopicObjectFactory`, so should work without the type as well.